packages/gatsby-source-drupal/src/utils.js

```diff
@@ -113,35 +94,39 @@ const handleReferences = (
     })
 
     delete node.drupal_relationships
-    referencedNodesLookup.set(node.id, referencedNodes)
+    cache.set(makeRefNodesKey(node.id), referencedNodes)
```

Comment:
```suggestion
    await cache.set(makeRefNodesKey(node.id), referencedNodes)
```
Do we need to await here as well?